src/index.ts

```diff
@@ -332,6 +346,8 @@ export default class FiatConnectClient implements FiatConectApiClient {
         `${this.config.baseUrl}/accounts/${params.fiatAccountId}`,
         {
           method: 'DELETE',
+          headers: this._getAuthHeader(),
+          ...this._getAuthHeader(),
```

Comment:
why do you need both here? shouldn't it just be `headers: this._getAuthHeader(),` like the others?

Reply:
oops, this got added by accident. Updated